src/queryTables.ts

```diff
@@ -73,52 +75,43 @@ async function populateTables (db: DatabaseType, usedTables: Array<string>, head
   const filteredTableDefinition = schema.filter(tableDefinition => usedTables.includes(tableDefinition.name))
 
   const promises = filteredTableDefinition.map(async (tableDefinition) => {
-    createTable(db, tableDefinition)
-
     const data = await fetchTableData(tableDefinition, headers)
 
-    if (data.length === 0) return
+    const fixedData = data.map(field => flattenObject(field, '_'))
+
+    if (fixedData.length === 0) return
+
+    const dynamicDefinition = {
+      ...tableDefinition,
+      data: fixedData[0] // We base table definition on first row
```

Comment:
loool, is back! The only problem with this approach is polymorphic endpoints. For instance, `/employees` will show more or less fields depending on the employee.

Reply:
This is already handled. I explicitly include all rows columns and normalize their shape. The problem of this approach is that it won't work when the request will be returning nothing (`[]`). So in this case I won't be able to know which is the schema :(